ert_shared/dark_storage/common.py

```diff
@@ -18,6 +18,18 @@ def ensemble_parameters(ensemble_name: str) -> List[dict]:
     ]
 
 
+def ensemble_parameter_names(ensemble_name: str) -> List[str]:
+    res = get_res()
+    return [
+        key
+        for key, parameter in (
+            (key, res.gather_gen_kw_data(ensemble_name, key))
+            for key in res.all_data_type_keys()
+            if res.is_gen_kw_key(key)
```

Comment:
This is fine but a bit wasteful IMO, I think we can extend a bit the libres facade to return exactly the `gen_kw_keys` maybe something like:
```python 
    def get_gen_kw_keys(self):
        """:rtype: list"""
        return self._enkf_main.getKeyManager().genKwKeys()
```